internal/pkg/scorecard/scorecard.go

```diff
@@ -336,6 +60,9 @@ func runTests() ([]scapiv1alpha1.ScorecardOutput, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to list files in %s/bin: %v", pluginDir, err)
 	}
+	if len(files) == 0 {
+		return nil, fmt.Errorf("No scorecard tests found in %s/bin", pluginDir)
```

Comment:
With a just-scaffolded project and a user that has not read the docs and configured a plugin, this will be the error they get, right? Maybe add the docs link here as well?

Reply:
They'd probably get the "Plugin directory not found" error that occurs higher up in the function. Either way, any error that occurs in this function gets handled by https://github.com/operator-framework/operator-sdk/pull/1472/files#diff-972e6cd1e836a6eed2c842356fd9ac78R112, which prints a link to the scorecard documentation.

Reply:
Ahh yeah, duh. Even better. Thanks for explaining.